src/Analysis/GWASApp/Components/Diagrams/PhenotypeHistogram/PhenotypeHistogram.jsx

```diff
@@ -145,16 +154,17 @@ const PhenotypeHistogram = ({
             <div className='GWASUI-column'>
               <InputNumber
                 id='input-minOutlierCutoff'
+                disabled={readOnly}
                 value={minOutlierCutoff}
                 onChange={(value) => {
                   setMinOutlierCutoff(value);
                   handleChangeMinOutlierCutoff(value);
                 }}
-                min={data.bins[0]?.start || 0}
+                min={(data.bins[0]?.start ?? 0) - 1}
```

Comment:
I'm not able to input a negative number for the Minimum outlier cutoff, even when the available range should allow for a negative value. It might be because the input doesn't allow for the ```-``` symbol as an input? On line 173 in the onKeyDown you might be able to just add ```&& key !== '-'```

Example: unable to input -1 here, but can enter zero and then click the decrement arrow button or press the down arrow to reach -1 

<img width="981" alt="image" src="https://github.com/user-attachments/assets/1eb14f55-d1b0-4267-8fc9-08275276e10a" />

Reply:
good idea for `&& key !== '-'` 

Done.